include/mo.php

```diff
@@ -85,7 +85,7 @@ public function import_from_db( $lang ) {
 		foreach ( $strings as $msg ) {
 			$entry = $this->make_entry( $msg[0], $msg[1] );
 
-			if ( '' !== $entry->singular ) {
+			if ( '' !== $entry->singular && '' !== $msg[1] ) {
```

Comment:
Let's improve the code and check `$msg[0]` and `$msg[1]` emptiness *before* we create the entry.

Reply:
🤔Not necessarily. `MO:make_entry()` empties entry singular property if the source string is empty
https://github.com/WordPress/WordPress/blob/6.7.1/wp-includes/pomo/mo.php#L329

Reply:
Done in 9394c3bbdc2db954690b59aa15c977caf5c4dca3